openstack/clientconfig/requests.go

```diff
@@ -656,6 +656,9 @@ func v3auth(cloud *Cloud, opts *ClientOpts) (*gophercloud.AuthOptions, error) {
 			} else if cloud.AuthInfo.DomainName != "" {
 				scope.DomainName = cloud.AuthInfo.DomainName
 			}
+			if cloud.AuthInfo.SystemScope != "" {
```

Comment:
@ignatenkobrain 
would you mind to add `SystemScope` into section above to build that option also from ENV and have it all covered? :)
```
       if cloud.AuthInfo.SystemScope == "" {
               if v := env.Getenv(envPrefix + "SYSTEM_SCOPE"); v != "" {
                       cloud.AuthInfo.SystemScope = v
               }
       }
```

Reply:
@ignatenkobrain really need this fix, could you push proposed changes?

Reply:
@ZdenekPesek can I reopen this PR with those changes?

Reply:
yes please, we sadly don't have much time/skills to push this through.